udata/harvest/backends/base.py

```diff
@@ -256,7 +256,7 @@ def process_dataset(self, remote_id: str, **kwargs):
             ]
             self.save_job()
 
-    def is_done(self) -> bool:
```

Comment:
We'll need to update [ckan backend](https://github.com/opendatateam/udata-ckan/blob/b7cf3c680ca643eff8bb9d5ecf58dad25a05d721/udata_ckan/harvesters.py#L127)